editorconfig.el

```diff
@@ -418,12 +418,15 @@ FILETYPE should be s string like `\"ini\"`, if not nil or empty string."
 (defun editorconfig-call-editorconfig-exec ()
   "Call EditorConfig core and return output."
   (let ((filename (buffer-file-name)))
-    (with-temp-buffer
-      (setq default-directory "/")
-      (if (eq 0
-              (call-process editorconfig-exec-path nil t nil filename))
-          (buffer-string)
-        (error (buffer-string))))))
+    (if (and (file-name-absolute-p filename)
+             (file-exists-p filename))
```

Comment:
This check is not good when creating a new file...
What is the value of `buffer-file-name` in this case?

Reply:
Right, maybe we can just pass `(expand-file-name filename)` to editorconfig(1) given that it doesn't care if the file actually exists.